### ISSUE TITLE
gender_culture_diverse_name returns nothing on some inputs

### ISSUE THREAD

**Issue by sebastianGehrmann** [GEM-benchmark/NL-Augmenter#297]
@XudongOliverShen 

I just tried your transformation on some summarization inputs and it returns nothing there. I believe there may be an issue if a document does not include any entities.

**Comment by XudongOliverShen** [comment]
That is true. I thought it's better to return nothing, instead of the input sentence. If returning the input sentence, it might confuse that the output has been transformed.

If you think returning the input sentence is more desirable, I can quickly code it up.
